graphql-test/src/main/java/org/jahia/test/graphql/GraphQLNodeTypesTest.java

```diff
@@ -401,7 +401,7 @@ public void shouldRetrieveIncludedAndNotExcludedNodeTypes() throws Exception {
         JSONArray nodeTypes = result.getJSONObject("data").getJSONObject("jcr").getJSONObject("nodeTypes").getJSONArray("nodes");
 
         NodeTypeRegistry nodeTypeRegistry = NodeTypeRegistry.getInstance();
-        Assert.assertEquals(11, nodeTypes.length());
+        Assert.assertEquals(10, nodeTypes.length());
```

Comment:
To improve stability just assert that there is results in the array, we found something.
Next loop make sure that what we found matches the criteria. 
How many results is not the matter in this test, otherwise we should ensure which 10 we found